src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/DynamicallyAccessedMemberTypes.cs

```diff
@@ -87,6 +87,11 @@ enum DynamicallyAccessedMemberTypes
         /// </summary>
         NonPublicEvents = 0x1000,
 
+        /// <summary>
+        /// Specifies all interfaces implementations
```

Comment:
Is `implementations` the right word to use here? I immediately thought this means the members that implement an interface, but not necessarily the interfaces themselves. The intention is that this allows for calling `.GetInterfaces()`. How about

```suggestion
        /// Specifies all interfaces implemented by the type.
```

(nit) the others all have a `.` at the end.